apps/challenges/views.py

```diff
@@ -3801,7 +3813,6 @@ def create_or_update_github_challenge(request, challenge_host_team_pk):
                     ).first()
                     if (
                         challenge_test_annotation_file
-                        and not challenge_phase.annotations_uploaded_using_cli
```

Comment:
What was this thing again? Can you please elaborate why this change was needed?

Reply:
Adding the challenge phase split after creation of challenge gives a error NoneType for annotations_uploaded_using_cli. we do not check for this in the creation code, there is another check below this if statement which does the opposition and check if it exist or not for annotations_uploaded_using_cli